jerry-main/main-unix.c

```diff
@@ -418,6 +418,20 @@ check_usage (bool condition, /**< the condition that must hold */
   }
 } /* check_usage */
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * The alloc function passed to jerry_create_instance
+ */
+static void *
+instance_alloc (size_t size,
+                void *cb_data_p __attribute__((unused)))
+{
+  return malloc (size);
```

Comment:
No. But if the jerry-libc is on, it will throw error when building, just like the case of `--jerry-libc=off --system-allocator=on`.

I can add a check in `main-unix.c`